category.html

```diff
@@ -51,6 +51,26 @@
     <title>Download</title>
   </head>
   <body>
+    <div class="wrapper">
+      <header>
+        <img src="assets/imgs/logo.svg" alt="app-logo" id="logo-img">
+        <p>The home of wallpapers</p>
+      </header>
+      <div class="nav">  <!---- for responsive ----->
+        <button class="nav-btns" type="submit" onclick="location.href='index.html'">Home</button>
+        <button class="nav-btns" type="submit" onclick="location.href='generate.html'">Generate</button>
+        <button class="nav-btns" type="submit" onclick="location.href='favorites.html'">Favorites</button>
```

Comment:
There is no need to give `type="submit"` to your button because it is by default
Also you haven't a form in your page